draft-ietf-taps-impl.md

```diff
@@ -413,11 +413,13 @@ Staggered racing can be used whenever a single node of the tree has multiple chi
 
 Staggered racing attempts occur in parallel. Implementations should not terminate an earlier child connection attempt upon starting a secondary child.
```

Comment:
I think this needs to be "Staggered racing attempts may occur in parallel" considering that the part that now follows right after in the text is the case when they do not occur in parallel.

Reply:
Is this clearer: ""Staggered racing attempts can proceed in parallel"?